File: 백준/Bronze/9366. 삼각형 분류/삼각형 분류.py
T = int(input())
for i in range(1, T+1):
    A, B, C = map(int,input().split())
    if sum([A, B, C]) <= 2*max(A, B, C):
        print(f'Case #{i}: invalid!')
    elif A == B == C:
        print(f'Case #{i}: equilateral')
    elif A == B or B == C or C == A:
        print(f'Case #{i}: isosceles')
    else:
        print(f'Case #{i}: scalene')

File: 백준/Bronze/9366. 삼각형 분류/삼각형 분류.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    static BufferedReader rd = new BufferedReader(new InputStreamReader(System.in));
    static StringBuilder sb = new StringBuilder();
    static StringTokenizer tk = null;

    static int T;
    static int A, B, C;
    static int max;

    public static void main(String[] args) throws Exception {
        T = Integer.parseInt(rd.readLine());

        for (int i=1; i<=T; i++) {
            tk = new StringTokenizer(rd.readLine());
            A = Integer.parseInt(tk.nextToken());
            B = Integer.parseInt(tk.nextToken());
            C = Integer.parseInt(tk.nextToken());

            max = Math.max(A, Math.max(B, C));

            if (A + B + C - max <= max) {
                sb.append("Case #").append(i).append(": invalid!\n");
            }
            else if (A == B && B == C) {
                sb.append("Case #").append(i).append(": equilateral\n");
            }
            else if (A == B || B == C || C == A) {
                sb.append("Case #").append(i).append(": isosceles\n");
            }
            else {
                sb.append("Case #").append(i).append(": scalene\n");
            }
        }
        System.out.println(sb);
    }
}
